core-builder/Dockerfile

```diff
@@ -1,24 +1,33 @@
 FROM openjdk:13.0.1-jdk-buster
 
-ENV PYTHON_VERSION "3.8.10"
+ENV PYTHON_VERSION="3.8.10"
+
+RUN printf '%s\n' \
```

Comment:
Ensure I could use this old `openjdk:13.0.1-jdk-buster` to allow _gradual upgrade of this image_.

Ideally we should change the base image, but I have to understand what to test apart from the ./release and operator helm scripts